src/frontend/src/common/redux/types.ts

```diff
@@ -9,7 +9,7 @@ export type ComplexActionType<T, U> = (payload: T) => {
 export type ActionType<T> = ComplexActionType<T, T>;
 
 export enum Pathogen {
-  COVID = "covid",
+  COVID = "SC2",
```

Comment:
Is there a reason we don't match the BE name entirely? I definitely see downsides, either way, but it feels slightly better to me to standardize the BE, FE, and Split flags all on a single name for each pathogen. Any name mismatch confusion around this anywhere in the chain seems pretty bad so that's why I'm thinking this.

Reply:
SC2 is the backend name, and also the split name, I believe.